src/poliastro/coordinates.py

```diff
@@ -130,3 +131,36 @@ def inertial_body_centered_to_pqw(r, v, source_body):
     v_pqw = (np.array([-sin(nu), (ecc + cos(nu)), 0]) * sqrt(k / p)).T * u.km / u.s
 
     return r_pqw, v_pqw
+
+
+def change_frame(orbit, frame_orig, frame_dest):
+    """Transforms Orbit from one frame to another.
+    
+    Parameters
+    ----------
+    orbit : ~poliastro.bodies.Orbit
+        Orbit to transform
+    frame_orig : ~astropy.coordinates.BaseRADecFrame
+        Initial frame
+    frame_dest : ~astropy.coordinates.BaseRADecFrame
+        Final frame
+
+    Returns
+    -------
+    orbit: ~poliastro.bodies.Orbit
+        Orbit in the new frame
+
+    """
+
+    orbit_orig = frame_orig(x=orbit.r[0], y=orbit.r[1], z=orbit.r[2],
+                            v_x=orbit.v[0], v_y=orbit.v[1], v_z=orbit.v[2],
+                            representation=CartesianRepresentation,
+                            differential_type=CartesianDifferential)
+
+    orbit_dest = orbit_orig.transform_to(frame_dest(obstime=orbit.epoch))
+    orbit_dest.representation = CartesianRepresentation
+
+    return Orbit.from_vectors(orbit.attractor,
+                              [orbit_dest.x, orbit_dest.y, orbit_dest.z] * u.km,
```

Comment:
I know you copied this from the example, but can you try using `orbit_dest.xyz` instead? http://docs.astropy.org/en/stable/api/astropy.coordinates.CartesianRepresentation.html#astropy.coordinates.CartesianRepresentation.xyz

Reply:
'GCRS' object has no attribute 'xyz'

Reply:
Sorry, it should be `orbit_dest.data.xyz`.